vault/audited_headers.go

```diff
@@ -37,7 +41,7 @@ func (a *AuditedHeadersConfig) add(header string, hmac bool) error {
 	a.Lock()
 	defer a.Unlock()
 
-	a.Headers[header] = &auditedHeaderSettings{hmac}
+	a.Headers[strings.ToLower(header)] = &auditedHeaderSettings{hmac}
```

Comment:
I believe audited headers setup is run as part of post unseal so it should always be lowercase after a switchover. LMK if you think that's not correct.